server/src/main/java/io/crate/metadata/blob/BlobSchemaInfo.java

```diff
@@ -58,8 +57,11 @@ public TableInfo getTableInfo(String name) {
                 name,
                 n -> blobTableInfoFactory.create(new RelationName(NAME, n), clusterService.state())
             );
-        } catch (ResourceUnknownException e) {
-            return null;
+        } catch (Exception e) {
```

Comment:
Why is the change here necessary?

Reply:
Because ResourceUnknownException is an interface now.